knowledge-base/grid-kb-specific-width.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Horizontal Grid Scrolling without Column Width
+description: How to Avoid Shrink or Disappear of Grid Columns and Get Horizontal Scrollbar?
```

Comment:
```suggestion
description: How to avoid shrinking or disappearing of Grid columns that have no column widths? How to get a horizontal scrollbar when the Grid columns have no widths?
```